### PR TITLE
don't log 404 to sentry

### PR DESCRIPTION
**Pull request by nicksnyder**
It is spamming sentry

image

image

### CODE REVIEW

File: pkg/trace/httptrace.go [hunk 1]
```diff
@@ -138,7 +138,7 @@ func Middleware(next http.Handler) http.Handler {
 		)
 
 		// If status code is not 2xx, notify Sentry
-		if m.Code/100 != 2 && m.Code/100 != 3 {
+		if m.Code/100 != 2 && m.Code/100 != 4 && m.Code/100 != 3 {
```
Comment (felixfbecker) [inline pkg/trace/httptrace.go]:
```suggestion
		if m.Code/100 != 2 && m.Code/100 != 3 && m.Code/100 != 4 {
```
Sorry

File: pkg/trace/httptrace.go [hunk 2]
```diff
@@ -138,7 +138,7 @@ func Middleware(next http.Handler) http.Handler {
 		)
 
 		// If status code is not 2xx, notify Sentry
```
Comment (slimsag) [inline pkg/trace/httptrace.go]: Update comment? Wildly inaccurate as it stands

File: pkg/trace/httptrace.go (hunk 1 above)
Comment (felixfbecker) [reply pkg/trace/httptrace.go]: Also, being used to JS: really interesting way to test this

File: pkg/trace/httptrace.go (hunk 2 above)
Comment (nicksnyder) [reply pkg/trace/httptrace.go]:
```suggestion
		// Notify sentry if the status code indicates our system had an error (e.g. 5xx).
```

**Pull request merged**